src/manifolds/Tucker.jl

```diff
@@ -0,0 +1,876 @@
+
+@doc raw"""
+Tucker{N, R, D, 𝔽} <: AbstractManifold{𝔽}
+
+The manifold of $N_1 \times \dots \times N_D$ real-valued or complex-valued tensors of
+fixed multilinear rank $(R_1, \dots, R_D)$ . If $R_1 = \dots = R_D = 1$, this is the
+manifold of rank-1 tensors.
+
+# Representation in HOSVD format
+
+Any tensor $\mathcal{A}$ on the Tucker manifold can be represented in HOSVD
+[^DeLathauwer2000] form
+```math
+\mathcal{A} = (U_1,\dots,U_D) \cdot \mathcal{C}
+```
+where $\mathcal C \in \mathbb{F}^{R_1 \times \dots \times R_D}$ and, for $d=1,\dots,D$,
+the matrix $U_d \in \mathbb{F}^{N_d \times R_d}$ contains the singular vectors of the
+$d$th unfolding of $\mathcal{A}$
```

Comment:
At the end of this line a . is missing.

I think it would be good to mention, that F is either the real-valued or the complex-valued numbers (or even quaternions?)